src/jobs/optimize-image.ts

```diff
@@ -148,6 +149,17 @@ const optimizeImage = async (job: Job) => {
         data: { url: urls },
       });
     } else if (model === "artistAvatar") {
+      artistFavicon = ico.sharpsToIco([sharp(buffer)], "artist_avatar.ico", {
+        sizes: [48],
+        resizeOptions: {},
+      });
+      logger.info("Uploading artist avatar favicon to bucket");
+      const faviconFinalName = `${destinationId}_favicon.ico`;
+      await minioClient.putObject(
+        finalMinioBucket,
+        faviconFinalName,
+        artistFavicon
+      );
```

Comment:
@simonv3 I've tried to store the favicon in Minio as you suggested. Let me know if I got it right :sweat_smile: :pray: